cli/cmd/install.go

```diff
@@ -19,6 +20,7 @@ import (
 	consts "github.com/linkerd/linkerd2/pkg/k8s"
 	"github.com/linkerd/linkerd2/pkg/tls"
 	"github.com/linkerd/linkerd2/pkg/version"
+	"github.com/pothulapati/mergo"
```

Comment:
The custom transformer logic requires us to create a new type which looks like bool, and make it be used everywhere as we can't add transformers for native types. 
This would impose a constraint on users to use the same which would be a hard requirement :\

Reply:
Understood. We'll have to move your fork into the linkerd org, like we did for the Stern fork.